e2e_test/batch/functions/concat.slt

```diff
@@ -0,0 +1,43 @@
+query T
+select concat('a', 'b');
+----
+ab
+
+query T
+select concat(NULL);
+----
+
+
+query T
+select concat(NULL, NULL);
+----
+
```

Comment:
https://www.sqlite.org/sqllogictest/doc/trunk/about.wiki
> Empty strings are rendered as "(empty)".

seems like `sqllogictest-rs` differs from spec for empty string. Need to verify.

Reply:
Yes result from `select concat(NULL, NULL);` is like so:
```
dev=> select concat(NULL) = '';
 ?column? 
----------
 t
(1 row)
```